docs/docs/preprocessing-external-images.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Preprocessing External Images
+---
+
+Gatsby allows powerful image processing features through using the [`Sharp`](https://github.com/lovell/sharp/) library to automatically process images to be performant and have features like lazy-loading. That said, this only works if the image is a file node in the GraphQL layer.
+
+If you want the same ecosystem for files that are hosted online and not located in your repo, [`gatsby-source-filesystem`](/packages/gatsby-source-filesystem/) has an API called `createRemoteFileNode` to solve this.
+
+This guide will walk you through how to use `createRemoteFileNode` process and get the same benefits of gatsby-transformer-sharp with externally sourced images.
+
+## Setup
+
+A usecase that this technique can be useful is if you want to create a featured image in a blogpost, but the image is a url up on the web instead of a local file. This could be hosted somewhere like Imgur, S3, or anywhere up on the internet.
+
+Give a sample post:
+
+```markdown
+---
+title: My first blogpost!
+featuredImgUrl: https://images.unsplash.com/photo-1560237731-890b122a9b6c
+---
+
+Hello World
+```
+
+We can have a frontmatter field for the url of the featured image we want to pull down and use as part of the site.
```

Comment:
Should change all `we` to `you`? Docs Style Guide suggest `you`
```suggestion
You can have a frontmatter field for the url of the featured image we want to pull down and use as part of the site.
```

Reply:
Done